functions/api/openocean/index.tsx

```diff
@@ -0,0 +1,63 @@
+const proOrigin = 'https://open-api-pro.openocean.finance/v4';
+
+const allowedEndpoints = ['reverseQuote', 'quote', 'swap', 'gasPrice'];
+
+const vaults = {
+  sei: '0x773B75CfB146bd5d1095fa9d6d45637f02B05119',
+  celo: '0x8cE318919438982514F9f479FDfB40D32C6ab749',
+  tac: '0xBBAFF3Bf6eC4C15992c0Fb37F12491Fd62C5B496',
+};
+
+export const onRequestGet: PagesFunction = async ({ request, env }) => {
+  const apikey = (env as any)['OPENOCEAN_APIKEY'];
```

Comment:
small: You can define an Env interface instead of bypassing using `as any`.
as in 
```
interface Env {
  OPENOCEAN_APIKEY: string;
  VITE_NETWORK?: string;
}
```